debian/autobake-deb.sh

```diff
@@ -19,6 +19,7 @@ set -e
 if [[ $TRAVIS ]]
 then
   export DEB_MTR_OPTIONS="--skip-test-list=${TRAVIS_BUILD_DIR}/.travis-skip-tests"
+  export DEB_BUILD_OPTIONS="parallel=4"
```

Comment:
.travis.yml has
env DEB_BUILD_OPTIONS="parallel=4" debian/autobake-deb.sh;

Didn't it work?